eng/pipelines/tools-repo-versioning.yml

```diff
@@ -16,7 +16,24 @@ jobs:
         # Remove Organization name i.e Transform Azure/azure-sdk-tools -> azure-sdk-tools before appending build Number
         $Tag = "$(Build.Repository.Name)".SubString("$(Build.Repository.Name)".IndexOf('/') + 1) + "_$(Build.BuildNumber)"
         Write-Host "Tagging Repo with $Tag"
+        echo "##vso[task.setvariable variable=Tag]$Tag"
         git -c user.name="azure-sdk" -c user.email="azuresdk@microsoft.com" tag -a $tag -m "Release tools version $Tag"
         git push $PushURL $tag
       displayName: Create and Push Version Tag
-      workingDirectory: $(Build.SourcesDirectory)
\ No newline at end of file
+      workingDirectory: $(Build.SourcesDirectory)
+
+    - template: /eng/pipelines/templates/steps/ref-updater.yml
+      parameters:
+        Tag: $(Tag)
+        ToolRepo: azure-sdk-tools
+        ToolsRepoPath: $(Build.SourcesDirectory)
+        Repos:
+#          - azure-sdk-for-android
```

Comment:
Are you planning to follow-up on this list or is it already the complete list of repo's referencing our tools repo?

Reply:
I added it to the issue https://github.com/Azure/azure-sdk-tools/issues/659
first I need to extend the tool for the other 5 repo. Then I will consolidate the list.